README.md

```diff
@@ -14,11 +14,11 @@ When initialized, the extension checks if the necessary binaries are present in
 
 > If this installation fails, you can run the installer manually, using the installer found at https://github.com/crc-org/macadam/releases/tag/v0.1.1. After this, you need to restart the extension which should find and use the binaries.
 
-### On Linux/x86_64
+### On Fedora Linux/x86_64
 
-The `macadam` binary must be installed in the directory `/opt/macadam/bin/`: download the binary from https://github.com/crc-org/macadam/releases/tag/v0.1.1 and rename it `macadam`.
+The `macadam` binary must be installed in the directory `/opt/macadam/bin/`: download the binary from https://github.com/crc-org/macadam/releases/tag/v0.1.1, rename it `macadam` and make it executable.
```

Comment:
If you want to add an example showing how to do this:
```
curl -L -o macadam https://github.com/crc-org/macadam/releases/download/v0.1.1/macadam-linux-amd64
chmod 755 macadam
sudo mv macadam /opt/macadam/bin/
```

Reply:
maybe `https://github.com/crc-org/macadam/releases/latest` rather than `https://github.com/crc-org/macadam/releases/tag/v0.1.1`
so it's always pointing to the latest binary

Reply:
the problem is that there is no equivalent for the curl command